internal/provider/data_source_aci_endpoint_security_group.go

```diff
@@ -8,6 +8,7 @@ import (
 	"context"
 	"fmt"
 
+	customtypes "github.com/CiscoDevNet/terraform-provider-aci/v2/internal/custom_types"
```

Comment:
should we use camelCase?
```suggestion
	customTypes "github.com/CiscoDevNet/terraform-provider-aci/v2/internal/custom_types"
```

Reply:
yes, will change that